app/api/weather.js

```diff
@@ -0,0 +1,33 @@
+import {get} from './request';
+
+
+const WEATHER_GOV_URL = 'https://api.weather.gov/points/';
+
+function parseForecastResult(name, lat, lon, forecastResult) {
+    const weather = {
+        'periods': forecastResult.properties.periods
+    };
+    weather[name] = {
+        name: name,
+        lat: Number(lat),
+        lon: Number(lon),
+        periods: forecastResult.properties.periods,
+    };
+    return weather;
+}
+
+export function getWeatherForecast(name, lat, lon) {
+  /* Get the Geonames result at a particular spot */
```

Comment:
this isn't actually using Geonames, is it? looks like you're able to go straight to weather.gov

Reply:
whoops! yeah, i'll fix